tools/gh-release.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+
+"""
+Draft a release in GitHub by calling into its API.
+
+Assumes all the releases are in the current path.
+"""
+
+import argparse
+import json
+import subprocess
+import tarfile
+from pathlib import Path
+
+from github import Github
+
+
+def handler_github_release(event, context):
+    print(json.dumps(event))
```

Comment:
leftover debugging statement?

Reply:
Kinda, this was initially going to be a Lambda, and I usually put that as the 1st line of every Lambda for sanity checking.

Reply:
Should be gone now.

Reply:
Ohh, I see, I was just concerned because one of the keys in that dictionary was called `GITHUB_KEY` or something, so thought we might not want to print that 😅